### ISSUE TITLE
get_peer_status, list out of range

### ISSUE THREAD

**Issue by martzuk** [20c/pybird#2]
```
>>> peer_state = pybird.get_peer_status()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/pybird.py", line 281, in get_peer_status
    peers = self._parse_peer_data(data=data, data_contains_detail=True)
  File "/usr/local/lib/python2.7/dist-packages/pybird.py", line 309, in _parse_peer_data
    peer_summary = self._parse_peer_summary(line)
  File "/usr/local/lib/python2.7/dist-packages/pybird.py", line 364, in _parse_peer_summary
    if ':' in elements[5]:
IndexError: list index out of range
```

**Comment by grizz** [comment]
@martzuk I believe this should be fixed in 1.1.0 - if not could you paste the bird output you're getting from the raw command?

**Comment by martzuk** [comment]
Seems to be resolved, thanks @grizz!
`>>> from pybird import PyBird
>>> pybird = PyBird(socket_file='/run/bird.ctl')     
>>> peer_state = pybird.get_peer_status()
>>> print peer_state
[{u'export_updates_received': 0, 'protocol': u'BGP', 'description': u'edge01.xxx02', u'import_updates_ignored': 0, u'import_withdraws_rejected': 0, u'export_updates_accepted': 0, u'import_updates_accepted': 0, u'import_updates_received': 0, u'export_updates_filtered': 0, 'address': u'x.x.x.x', u'import_withdraws_ignored': 0, 'routes_imported': 0, 'asn': u'65500', u'export_updates_rejected': 0, u'import_updates_filtered': 0, u'export_withdraws_accepted': 0, u'import_withdraws_accepted': 0, u'export_withdraws_received': 0, 'up': False, 'name': u'b4_edge01_xxx02', 'state': u'Passive', 'routes_exported': 0, u'import_updates_rejected': 0, 'last_change': datetime.datetime(2017, 2, 24, 0, 0), u'import_withdraws_received': 0}, {u'export_updates_received': 0, 'protocol': u'BGP',
[SNIPPED]`
